runtime/shibuya/src/lib.rs

```diff
@@ -1688,7 +1688,7 @@ pub type Executive = frame_executive::Executive<
 pub type Migrations = (Unreleased, Permanent);
 
 /// Unreleased migrations. Add new ones here:
-pub type Unreleased = (pallet_dapp_staking::migration::DappStakingCleanupMigration<Runtime>,);
```

Comment:
Same for Shibuya, runtime version is still **1301**. We need to keep the cleanup.